src/tests/pools/D3M4626TypePool.t.sol

```diff
@@ -162,7 +157,11 @@ contract D3M4626TypePoolTest is D3MPoolBaseTest {
         assertEq(pool.maxWithdraw(), 1e18);
     }
 
-    function test_maxDeposit_returns_max_uint() public {
+    function invariant_pool_maxDeposit() public {
+        assertEq(pool.maxDeposit(), type(uint256).max);
+    }
+
+    function invariant_vault_maxDeposit() public {
         assertEq(pool.maxDeposit(), type(uint256).max);
```

Comment:
Should be `vault` here.

Reply:
done here 3615bb4b48372a63557e87d6649bccf16dc09f9b

Reply:
Can be resolved I think. @sunbreak1211